mmv1/products/dataplex/Glossary.yaml

```
@@ -0,0 +1,113 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Glossary'
+description: |
+  Represents a collection of categories and terms defined by the user. Glossary is a top level resource and is the GCP parent resource of all the categories and terms within it.
+base_url: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/glossaries?glossary_id={{glossary_id}}'
+update_verb: 'PATCH'
+update_mask: true
+timeouts:
+  insert_minutes: 5
+  update_minutes: 5
+  delete_minutes: 5
+autogen_async: true
+import_format: ['projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}']
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+iam_policy:
+  method_name_separator: ':'
+  parent_resource_attribute: 'glossary_id'
+  example_config_body: 'templates/terraform/iam/iam_attributes.go.tmpl'
+  import_format:
+    - 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
```

Looks like you need to specify a second item that's just glossary_id. This is weird, not documented, and isn't actually a valid import format; I'm going to open a ticket to fix that at the provider level but for now this is the standard workaround.

```suggestion
    - 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
    - '{{glossary_id}}'
```

---

created https://github.com/hashicorp/terraform-provider-google/issues/22667